packages/repo-doctor/src/index.js

```diff
@@ -44,7 +47,11 @@ const main = async () => {
     repositories = repositories.filter(repo => repo.slug === args.repo)
   }
 
-  const rules = setupRules(config, args)
+  let rules = setupRules(config, args)
+  if (args.rule) {
+    rules = rules.filter(r => r.constructor.name === args.rule)
```

Comment:
Could we use the name from the schema instead? Constructor names may differ with imports, exports, memoization or whatnot.

Reply:
Mmh the constructor is the name of the class and it will not change with imports and exports. Classes are not subject to memoization. So I don't think we have an issue here.